test/fixtures/all_passing/results.json

```diff
@@ -0,0 +1,24 @@
+{
+    "status": "pass",
+    "message": null,
+    "tests": [
+        {
+            "name": "first test",
+            "status": "pass",
+            "message": null,
+            "output": ""
```

Comment:
For the message, the [spec](https://github.com/exercism/automated-tests/blob/master/docs/interface.md) mentions to use to null or omit the key entirely when there is no error. We don't specify what to do with empty output. @iHiD would an empty string as output result in output being shown on the website? In other words: should this be `null` when it is empty?

Reply:
I missed that in the spec. I can easily set it to null

Reply:
I think `null` would make sense (it it not in the spec for the `output` field currently)

Reply:
It could be that the student wants to output something but it ends up being empty, so there's no (easy) way to tell whether there was no output or the output was empty.

Reply:
True

Reply:
`null` would be better but I don't think we show it if it's `null` or `""`

Reply:
Resolved by f95fa05